Assets/Scripts/Framework/Gameplay/MiniGames/Objects/TurningEnemy.cs

```diff
@@ -0,0 +1,69 @@
+using UnityEngine.Events;
+using UnityEngine;
+using TMPro;
+
+namespace Framework.Gameplay.MiniGames.Objects
+{
+    public sealed class TurningEnemy : MonoBehaviour
+    {
+        [SerializeField] private UnityEvent onCaught = new();
+        [HideInInspector] public bool minigameActive;
+        [SerializeField] private bool _isWatching;
+        private SpriteRenderer _spriteRenderer; //temporary until art implementation
+        private float _lookInterval;
+        private Timer timer;
```

Comment:
```suggestion
        private Timer _timer;
```
[_ _ _ _ _](https://github.com/bas-boop/ProjectIgnite/blob/main/CodeConvention.md#variables)